### ISSUE TITLE
Feedback from Karlos

### ISSUE THREAD

**Issue by KarlosMuradyan** [UBC-MDS/DSCI-532_gr202_r_dashboard#58]
Hi there,

I think simple, effective and pleasant sums up all the work that you have done for the dashboard. I really enjoyed the layout and the colors present in your dashboard. Overall, I would like to give feedback on the following 4 principles:

- **On-boarding experience**: It is really nice to see some introduction at the top that provides some insight into the data and possible research questions that can arise with it. There is also a small description of how it is possible to interact with the dashboard to have the best experience. How to filter? What to filter? Why filter? ... All these questions are answered in the introductory part of your dashboard!!
- **Build trust in your analysis**: You did a great job by providing links to your github repo and the data source. It means that your results are reproducible and everyone can use and get the same results as you did. This increases the trust in your analysis which is one of the most important things.
- **Use a consistent layout**: I cannot imagine a better layout than you did on your dashboard. Each subsection has separate colors but with a very good color scheme (dark blue, gray and white- amazing combination). All filters are on the left-hand side and are really intuitive. As I already mentioned, at the bottom you have links to github and data source. The only thing that I would suggest to change is the layout of the links. I would suggest to fit it in the one line above that so that it will look prettier and more consistent.
- **Allow users to filter data**: You had only three filters and very informative. With the help of diverse values in those filters, we can answer lots of research questions and examine in detail Boston city crime rates. We can create more hundreds of combinations and compare whatever we select and discard whatever is redundant for our study.

One thing to notice,  your map fails sometimes. For example, when I select Hyde Park, the map doesn't filter. If this is solved, the dashboard will look fantastic!!

To sum up, you really did a great job. All your plots are informative and help to answer the whole bunch of research questions by giving the flexibility to the user to filter as much as they want.
